### PR TITLE
Remove unnecessary replace statements in `go.mod`

### PR DESCRIPTION
**Pull request by berndverst**
# Description

Remove unnecessary replace statements in `go.mod`

### CODE REVIEW

File: go.mod
Comment (ItalyPaleAle) [inline go.mod:379]: Don't remove this

Comment (berndverst) [reply go.mod:379]: Why is this not upstream already? It is unacceptable to use @shivamkm07's personal fork for so many months. We need to find a different strategy.

Comment (berndverst) [reply go.mod:379]: Latest version is 1.4.1 and we are stuck on this unmaintained 1.3.6 fork.

Comment (berndverst) [reply go.mod:379]: Can you complete work on https://github.com/eclipse/paho.mqtt.golang/pull/578 @shivamkm07 ?

**Review by ItalyPaleAle — APPROVED** [review]

File: go.mod
Comment (shivamkm07) [reply go.mod:379]: @berndverst Updated the PR as requested. Waiting for the maintainers' response. Thanks!

**Pull request merged**
